src/main/java/io/fusionauth/jwt/ec/ECDSASignature.java

```diff
@@ -86,10 +86,17 @@ public byte[] derDecode(Algorithm algorithm) throws IOException {
     }
 
     int len = result.length / 2;
-    //noinspection ManualMinMaxCalculation
-    System.arraycopy(r, r.length > len ? 1 : 0, result, r.length < len ? 1 : 0, r.length > len ? len : r.length);
-    //noinspection ManualMinMaxCalculation
-    System.arraycopy(s, s.length > len ? 1 : 0, result, s.length < len ? (len + 1) : len, s.length > len ? len : s.length);
+
```

Comment:
Add a comment for these after they are cleaned up about offsetting the destination position for left padding when `r` or `s` is shorter than `66` bytes in the `ES512` case.

Do we know whether it _only_ happens with that algorithm? Or just that we have never seen it with another?

Reply:
It can occur in any algorithm in theory. In my testing, I could get `r` and `s` src and dst positions to be greater than `0` for all three algorithms.

Reply:
❤️